data/autoyast_sle15/autoyast_containers_s390x.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <addons config:type="list">
+      <addon>
+        <name>sle-module-basesystem</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-development-tools</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-server-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-containers</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
```

Comment:
```
localhost:~ # SUSEConnect --list                                                                                                                                                       [10/195]
AVAILABLE EXTENSIONS AND MODULES                                                                                                                                                               
                                                                                                                                                                                               
    Basesystem Module 15 SP2 x86_64 (Activated)                                                                                                                                                
    Deactivate with: SUSEConnect -d -p sle-module-basesystem/15.2/x86_64                                                                                                                       
                                                                                                                                                                                               
        Containers Module 15 SP2 x86_64                                                                                                                                                        
        Activate with: SUSEConnect -p sle-module-containers/15.2/x86_64                                                                                                                        
                                                                                                                                                                                               
        Desktop Applications Module 15 SP2 x86_64 (Activated)                                                                                                                                  
        Deactivate with: SUSEConnect -d -p sle-module-desktop-applications/15.2/x86_64                                                                                                         

            Development Tools Module 15 SP2 x86_64 (Activated)
            Deactivate with: SUSEConnect -d -p sle-module-development-tools/15.2/x86_64

            SUSE Linux Enterprise Workstation Extension 15 SP2 x86_64 (BETA)
            Activate with: SUSEConnect -p sle-we/15.2/x86_64 -r ADDITIONAL REGCODE 

        Python 2 Module 15 SP2 x86_64
        Activate with: SUSEConnect -p sle-module-python2/15.2/x86_64

        SUSE Cloud Application Platform Tools Module 15 SP2 x86_64
        Activate with: SUSEConnect -p sle-module-cap-tools/15.2/x86_64

        SUSE Linux Enterprise Live Patching 15 SP2 x86_64 (BETA)
        Activate with: SUSEConnect -p sle-module-live-patching/15.2/x86_64 -r ADDITIONAL REGCODE 

        SUSE Package Hub 15 SP2 x86_64
        Activate with: SUSEConnect -p PackageHub/15.2/x86_64

        Server Applications Module 15 SP2 x86_64 (Activated)
        Deactivate with: SUSEConnect -d -p sle-module-server-applications/15.2/x86_64

            Legacy Module 15 SP2 x86_64
            Activate with: SUSEConnect -p sle-module-legacy/15.2/x86_64

            Public Cloud Module 15 SP2 x86_64
            Activate with: SUSEConnect -p sle-module-public-cloud/15.2/x86_64

            SUSE Linux Enterprise High Availability Extension 15 SP2 x86_64 (BETA)
            Activate with: SUSEConnect -p sle-ha/15.2/x86_64 -r ADDITIONAL REGCODE 

            Web and Scripting Module 15 SP2 x86_64
            Activate with: SUSEConnect -p sle-module-web-scripting/15.2/x86_64

        Transactional Server Module 15 SP2 x86_64
        Activate with: SUSEConnect -p sle-module-transactional-server/15.2/x86_64
```

SDK depends on sle-module-desktop-applications

Reply:
Don't understand your comment. What do you mean?

Reply:
```
AVAILABLE EXTENSIONS AND MODULES                                                                                                                                                               
                                                                                                                                                                                               
    Basesystem Module 15 SP2 x86_64 (Activated)                                                                                                                                                
    Deactivate with: SUSEConnect -d -p sle-module-basesystem/15.2/x86_64                                                                                                                       
                                                                                                                                                                                               
        Containers Module 15 SP2 x86_64                                                                                                                                                        
        Activate with: SUSEConnect -p sle-module-containers/15.2/x86_64                                                                                                                        
                                                                                                                                                                                               
        Desktop Applications Module 15 SP2 x86_64 (Activated)                                                                                                                                  
        Deactivate with: SUSEConnect -d -p sle-module-desktop-applications/15.2/x86_64                                                                                                         

            Development Tools Module 15 SP2 x86_64 (Activated)
            Deactivate with: SUSEConnect -d -p sle-module-development-tools/15.2/x86_64

            SUSE Linux Enterprise Workstation Extension 15 SP2 x86_64 (BETA)
            Activate with: SUSEConnect -p sle-we/15.2/x86_64 -r ADDITIONAL REGCODE 
.
.
.
localhost:~ # SUSEConnect -d -p sle-module-desktop-applications/15.2/x86_64
Deregistering system from registration proxy http://all-201.1.proxy.scc.suse.de

Deactivating sle-module-desktop-applications 15.2 x86_64 ...
Error: Registration server returned 'Cannot deactivate the product "Desktop Applications Module". Other activated products depend upon it.' (422)
```

Reply:
I will leave as it is since it isn't complaining about dependencies, maybe autoyast is handling this automatically. Anyway, if we decide to change it, let's do it in a separate PR where we change in all 4 archs profiles.